INZFS/Pages/Index.cshtml

```diff
@@ -20,6 +20,7 @@
 
         <p class="govuk-body">The fund is open to UK organisations of any size, in both the public and private sector. It is particularly focused on helping small and medium-sized businesses, including start-ups. </p>
         <p class="govuk-body">Academic institutions are welcome to apply as part of a consortium. They can not be a lead or sole applicant.</p>
+        <p class="govuk-body">Sole traders are not eligible for the fund.’ under the paragraph about academic institutions please?</p>
```

Comment:
The punctuation on this line looks a little weird

Reply:
resolved